docker/compose/avalon-multi-pool.yaml

```diff
@@ -79,6 +81,8 @@ services:
         - https_proxy
         - no_proxy
         - ENCLAVE_TYPE=wpe
+        - IMAGE=$IMAGE
```

Comment:
Would rely on defaults from docker-compose.yaml. Double check for all other attributes as well.

Reply:
Removed all the duplicate attributes for the overriden service.